CMakeLists.txt

```diff
@@ -146,6 +146,9 @@ if(TIGL_USE_GLOG)
 endif(TIGL_USE_GLOG)
 
 # enable C++11 support if available
+# TODO(bgruber): enable when cmake 3.1 is available and remove the flagtests underneath
+#set(CMAKE_CXX_STANDARD 11)

```

Comment:
Please comment them in! Is cmake 3.1 not availble on all platforms in CI?

Reply:
I did not check this, but the file starts with `cmake_minimum_required (VERSION 2.8.12)`, so I assumed that is your minimum required version.